scripts/pre_e2e.sh

```diff
@@ -4,5 +4,18 @@ set -Eeuo pipefail
 # Generate big files which are used during testing
 # `uploadProject.spec.js` -> should reject files bigger than `MAX_FILE_SIZE`
 auto_gen_fixtures_path=e2e/cypress/fixtures/auto-gen
-mkdir $auto_gen_fixtures_path
-truncate -s "${MAX_FILE_SIZE}" "${auto_gen_fixtures_path}/big.txt"
+rm -rf $auto_gen_fixtures_path && mkdir $auto_gen_fixtures_path
+truncate -s "${MAX_FILE_SIZE:-6M}" "${auto_gen_fixtures_path}/big.txt"
```

Comment:
I would prefer to error if this is not set.

Reply:
Resolved in c8d2b99.